scripts/repair_batlas_references.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+import argparse
+import itertools
+import json
+import sys
+import transaction
+
+from pleiades.dump import getSite, spoofRequest
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Repair old-style and idiosyncratic BAtlas references on Place objects"
+    )
+    parser.add_argument("input_file", type=str, help="The JSON input file name")
+    parser.add_argument("--dry-run", action="store_true", help="Enable dry run mode")
+    # Deal with run script wrapping. We're passed a -c flag we don't use:
+    parser.add_argument("-c", type=str, help=argparse.SUPPRESS)
```

Comment:
What's this?

Reply:
The `bin/[instance]/run` wrapper adds a `-c` argument that results in an error without some form of intervention. This seemed better than modifying `sys.argv` or similar, but maybe there's another alternative?

```
usage: interpreter [-h] [--dry-run] input_file
interpreter: error: unrecognized arguments: -c var/full.json
```

Reply:
Ah, weird. Never noticed that before.